recipes/r-seasonal/meta.yaml

```
@@ -0,0 +1,86 @@
+{% set version = '1.7.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-seasonal
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/seasonal_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/seasonal/seasonal_{{ version }}.tar.gz
+  sha256: 29e8d7c7a01981f52cce5478a03b5ee5074add8af35452446dbd7e1221a0cd27
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [osx]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip  # [win]
+
+  host:
+    - r-base
+    - r-x13binary
+
+  run:
+    - r-base
+    - r-x13binary
+
+test:
+  commands:
+    - $R -e "library('seasonal')"           # [not win]
+    - "\"%R%\" -e \"library('seasonal')\""  # [win]
+
+about:
+  home: http://www.seasonal.website
+  license: GPL-2.0-or-later
+  summary: Easy-to-use interface to X-13-ARIMA-SEATS, the seasonal adjustment software by the
+    US Census Bureau. It offers full access to almost all options and outputs of X-13,
+    including X-11 and SEATS, automatic ARIMA model search, outlier detection and support
+    for user defined holiday variables, such as Chinese New Year or Indian Diwali. A
+    graphical user interface can be used through the 'seasonalview' package. Uses the
+    X-13-binaries from the 'x13binary' package.
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - rluria14
+    - ndmaxar
+    - RyanHackney
```

Please add the conda-forge/r team:
```suggestion
    - RyanHackney
    - conda-forge/r
```